WordPress/src/main/res/layout/item_pie_chart_legend.xml

```diff
@@ -12,9 +13,11 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center_horizontal"
-        android:drawablePadding="@dimen/margin_medium"
+        android:paddingEnd="@dimen/margin_small"
+        android:paddingStart="@dimen/margin_small"
```

Comment:
Just nitpicking, I like using `android:paddingHorizontal="@dimen/margin_small"` because it's one line shorter. 😄 Feel free to ignore.

Reply:
Agreed 👍  updated